id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoServiceImpl.java

```diff
@@ -785,41 +897,4 @@ protected byte[] convertToBytes(Object identity) throws IdRepoAppException {
 		}
 	}
 
-	private Map<String, HandleDto> checkAndGetHandles(IdRequestDTO request) throws IdRepoAppException {
```

Comment:
This should not be removed. This one is newly added, and this is not removed for resident.

Reply:
Added